### PR TITLE
Put common test code into one file

### PR DESCRIPTION
**Pull request by JanuszL**
Signed-off-by: Janusz Lisiecki <jlisiecki@nvidia.com>

### CODE REVIEW

**Comment by JanuszL** [comment]
Builds 35138

File: qa/test_template.sh [hunk 1]
```diff
@@ -0,0 +1,28 @@
+topdir=$(cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null && pwd )/..
+
+# Install dependencies
+# Note: glib-2.0 depends on python2, so reinstall the desired python afterward to make sure defaults are right
```
Comment (ptrendx) [inline qa/test_template.sh]: Do we actually need this library? This hacky handling of Python versions has potential to screw us at some point.

Comment (cliffwoolley) [reply qa/test_template.sh]: You can blame me for that particular bit of nonsense.  (It was already there long before this PR.)

Comment (ptrendx) [reply qa/test_template.sh]: Do you remember why this was introduced?

Comment (JanuszL) [reply qa/test_template.sh]: I would leave it now as it is. We can easily change it latter (as now it is only in one place).

Comment (cliffwoolley) [reply qa/test_template.sh]: @ptrendx - I don't know why the dependency is there.  It was there long before I came to it.  I  agree with Janusz though; we can address it separately.

**Pull request merged**